### PR TITLE
fix: DBus Null annotations

### PR DESCRIPTION
**Pull request by workgroupengineering**
## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021630-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.FreeDesktop/DBusIme/Fcitx/FcitxICWrapper.cs [hunk 1]
```diff
@@ -18,34 +18,37 @@ public FcitxICWrapper(IFcitxInputContext1 modern)
             _modern = modern;
         }
 
-        public Task FocusInAsync() => _old?.FocusInAsync() ?? _modern.FocusInAsync();
+        public Task FocusInAsync() => _old?.FocusInAsync() ?? _modern?.FocusInAsync() ?? Task.CompletedTask;
```
Comment (kekekeks) [inline src/Avalonia.FreeDesktop/DBusIme/Fcitx/FcitxICWrapper.cs:21]: One of `_old` or `_modern` is guaranteed to be non-null.

**Review by kekekeks — APPROVED** [review]
In general lgtm

File: Avalonia.sln [hunk 2]
```diff
@@ -38,6 +38,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{A689DEF5-D50F-4975-8B72-124C9EB54066}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		src\Shared\IsExternalInit.cs = src\Shared\IsExternalInit.cs
```
Comment (jmacato) [inline Avalonia.sln:41]: What is this file for?

File: src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj [hunk 3]
```diff
@@ -5,6 +5,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\Shared\IsExternalInit.cs" Link="IsExternalInit.cs" />
```
Comment (jmacato) [inline src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj:9]: This as well, i cant find it in the PR?

**Review by jmacato — DISMISSED** [review]
there's some that seems like it's autogenerated changes in the sln and proj files, please remove those.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021788-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: Avalonia.sln (hunk 2 above)
Comment (maxkatz6) [reply Avalonia.sln:41]: Adds a file to the solution tree

File: src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj (hunk 3 above)
Comment (maxkatz6) [reply src/Avalonia.FreeDesktop/Avalonia.FreeDesktop.csproj:9]: IsExternalInit is required for records

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021796-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Pull request merged**